### DISCUSSION TITLE
Updated curl commands in Docker files

### DISCUSSION THREAD

**Discussion by ewhauser** [renovatebot/renovate#15128] [Request Help]
We have a fair amount of curl commands that look this in our Dockerfile's:

```
 RUN curl -OL https://github.com/bazelbuild/bazelisk/releases/download/v1.11.0/bazelisk-linux-amd64 \
    && echo "231ec5ca8115e94c75a1f4fbada1a062b48822ca04f21f26e4cb1cd8973cd458 bazelisk-linux-amd64" | sha256sum -c - \
    && mv bazelisk-linux-amd64 /usr/bin/bazel \
    && chmod +x /usr/bin/bazel
```

Is there a way to do have Renovate update these? I read through the docs on https://docs.renovatebot.com/modules/manager/regex/ but it wasn't clear to me how to update both the version number and the SHA. I'm happy to rewrite the Dockerfile to use variables if that makes it easier to update.

Thanks!

**Comment by viceice** [comment]
should work if you have capture groups for current value and digest. github-releases datasource should be able to handle the digest updates for release attachments.
